lib/Traits/TraitsClasses/BaseMapTraits.ts

```diff
@@ -0,0 +1,65 @@
+import CatalogMemberFactory from "../../Models/CatalogMemberFactory";
+import modelReferenceTrait from "../Decorators/modelReferenceTrait";
+import objectArrayTrait from "../Decorators/objectArrayTrait";
+import primitiveArrayTrait from "../Decorators/primitiveArrayTrait";
+import primitiveTrait from "../Decorators/primitiveTrait";
+import ModelReference from "../ModelReference";
+import ModelTraits from "../ModelTraits";
+
+export class BaseMapTraits extends ModelTraits {
+  @primitiveTrait({
+    type: "string",
+    name: "Image",
+    description: "Path to the basemap image"
+  })
+  image?: string;
+
+  @modelReferenceTrait({
+    factory: CatalogMemberFactory,
+    name: "Base map item",
+    description: "Base map item definition"
```

Comment:
Could this please include a sentence about referencing existing catalog items (eg `"//Surface Geology"`)

Reply:
done